docs/quick-start.md

```diff
@@ -0,0 +1,180 @@
+- When you edit `vars.yml`, make sure to pick a correct domain, because you can't change the domain after deployment. If you pick a wrong one, you'll have to run the Uninstalling step.
+
+## Install
+
+<sup>This section is optimized for this quick-start guide and is derived from the following full-documentation page: [Installing](installing.md)</sup>
+
+After editing `vars.yml` and `hosts` files, let's start the **installation** procedure.
+
+Before installing, you need to update the Ansible roles in this playbook. To update them, run `just roles` on your local computer.
+
+Then, run the command below to start installation:
+
+````sh
+ansible-playbook -i inventory/hosts setup.yml --tags=install-all,ensure-matrix-users-created,start
+````
+
+If you **don't** use SSH keys for authentication, but rather a regular password, you may need to add `--ask-pass` to the command.
+
+If you **do** use SSH keys for authentication, **and** use a non-root user to *become* root (sudo), you may need to add `-K` (`--ask-become-pass`) to the command.
+
+Wait until the command completes. If it's all green, everything should be running properly.
+
+## Create your user account
+
+<sup>This section is optimized for this quick-start guide and is derived from the following full-documentation page: [Registering users](registering-users.md)</sup>
+
+As you have configured your brand new server and the client, you need to **create your user account** on your Matrix server.
+
+To register a user via this Ansible playbook, run the command below on your local computer.
+
+**Notes**:
+- Before running it, make sure to edit `YOUR_USERNAME_HERE` and `YOUR_PASSWORD_HERE`
+- In the command below, `YOUR_USERNAME_HERE` is just a plain username (like `john`), not your full `@user:example.com` identifier
+
+```sh
+ansible-playbook -i inventory/hosts setup.yml --extra-vars='username=YOUR_USERNAME_HERE password=YOUR_PASSWORD_HERE admin=<yes|no>' --tags=register-user
+
+# Example: ansible-playbook -i inventory/hosts setup.yml --extra-vars='username=john password=secret-password admin=yes' --tags=register-user
+```
+
+## Finalize server installation
+
+<sup>This section is optimized for this quick-start guide and is derived from the following full-documentation page: [Server Delegation](howto-server-delegation.md)</sup>
+
+Now you've configured Matrix services and your user account, you need to **finalize the installation process** by [setting up Matrix delegation (redirection)](howto-server-delegation.md), so that your Matrix server (`matrix.example.com`) can present itself as the base domain (`example.com`) in the Matrix network.
+
+**This is required for federation to work!** Without a proper configuration, your server will effectively not be part of the Matrix network.
+
+There are several ways to configure the delegation, but the easiest one is to [serve the base domain](configuring-playbook-base-domain-serving.md) from the integrated web server.
```

Comment:
It feels like this section focuses on the "base domain serving" option, which is nice and quick, but..

People who cannot afford to point the base domain to the playbook will need other options.

Above, we already link to `howto-server-delegation.md` twice, but maybe we should also be more explicit as to what your options are if you cannot serve the base domain on the Matrix server.

Perhaps a sentence like this can be added somewhere:

> If you're using the base domain for other purposes and cannot point it to the Matrix server (and thus cannot "serve the base domain" from it), you most likely need to [Manually install well-known files on the base domain's server](configuring-well-known.md#manually-installing-well-known-files-on-the-base-domains-server), but feel free to familiarize yourself with all [Server delegation (redirection)](howto-server-delegation.md) options.

-------

More restructuring could be done as well. For example (just a quick draft below):

> You have 2 options:
>
> - If you can afford to point the base domain at the Matrix server, follow the instructions below which guide you into [serving the base domain](configuring-playbook-base-domain-serving.md) from the Matrix server
> - Alternatively, if you're using the base domain for other purposes and cannot point it to the Matrix server (and thus cannot "serve the base domain" from it), you most likely need to [Manually install well-known files on the base domain's server](configuring-well-known.md#manually-installing-well-known-files-on-the-base-domains-server), but feel free to familiarize yourself with all [Server delegation (redirection)](howto-server-delegation.md) options.